python/ray/autoscaler/util.py

```diff
@@ -56,7 +56,7 @@ def validate_config(config: Dict[str, Any]) -> None:
     try:
         jsonschema.validate(config, schema)
     except jsonschema.ValidationError as e:
-        raise jsonschema.ValidationError(message=e.message) from None
+        raise e from None
```

Comment:
just for my understanding, this hides the rest of the stacktrace right?

Reply:
The `from None` does, I included the full error message to give slightly more context.